ambry-account/src/main/java/com/github/ambry/account/MySqlAccountService.java

```diff
@@ -53,6 +54,10 @@
 public class MySqlAccountService extends AbstractAccountService {
 
   private static final Logger logger = LoggerFactory.getLogger(MySqlAccountService.class);
+  private static final String SEPARATOR = "_";
```

Comment:
Can we change this to ":" ?  That char is less commonly (or never) used in account or container names